README.md

```diff
@@ -114,28 +130,8 @@ Tape consists of several workers that run in goroutines, so that the pipeline is
 
 ### log
```

Comment:
it's better to move "how to use log level" out of details.

Reply:
see 
someone using tape.
having issue.
report issue.
ask for enable with debug level...

vs
someone using tape.
having issue.
report with debug level log.

Reply:
good point, I'll update issue template to include this